spec.md

```diff
@@ -1074,6 +1058,16 @@ message ControllerServiceCapability {
       PUBLISH_UNPUBLISH_VOLUME = 2;
       LIST_VOLUMES = 3;
       GET_CAPACITY = 4;
+      CREATE_DELETE_SNAPSHOT = 5;
+      // For storage systems that need to upload the snapshot after
+      // it is cut, LIST_SNAPSHOTS is a required RPC because CO needs
+      // to call it to find out whether the upload is complete.
+      LIST_SNAPSHOTS = 6;
+      // Currently the only way to consume a snapshot is to create
+      // a volume from it. Therefore plugins supporting
+      // CREATE_DELETE_SNAPSHOT should also support
+      // CREATE_VOLUME_FROM_SNAPSHOT.
+      CREATE_VOLUME_FROM_SNAPSHOT = 7;
```

Comment:
If this is a requirement for `CREATE_DELETE_SNAPSHOT` why have it as a separate capability?

Reply:
You are right.  This is not necessary.  I'll remove this one and change the spec accordingly.